runtime/oti/jclprots.h

```diff
@@ -120,6 +120,11 @@ extern J9_CFUNC jlong JNICALL
 Java_com_ibm_lang_management_internal_ExtendedRuntimeMXBeanImpl_getProcessIDImpl(JNIEnv *env, jclass clazz);
 #endif /* JAVA_SPEC_VERSION < 19 */
 
+#if JAVA_SPEC_VERSION < 20
+extern J9_CFUNC void JNICALL
+Java_java_lang_Thread_stopImpl(JNIEnv *env, jobject rcv, jobject stopThrowable);
+#endif /* JAVA_SPEC_VERSION < 20 */
+
```

Comment:
This should appear in the group for `thread.cpp` (currently says `thread.c`).

Reply:
Moved to `thread.cpp` section, also removed `JVM_StopThread()`.